examples/05_glm_second_level/plot_oasis.py

```diff
@@ -105,7 +105,7 @@
 from nilearn.glm.second_level import SecondLevelModel
 
 second_level_model = SecondLevelModel(
-    smoothing_fwhm=2.0, mask_img=mask_img, n_jobs=2
+    smoothing_fwhm=2.0, mask_img=mask_img, n_jobs=2, minimize_memory=False
```

Comment:
Maybe check what is the impact on doc generation cost

Reply:
## second level example

### before

https://nilearn.github.io/dev/auto_examples/05_glm_second_level/plot_oasis.html#generating-a-report

![image](https://github.com/nilearn/nilearn/assets/6961185/3c46ac51-0002-43b7-aed2-af13987bf11b)

## after

https://output.circle-artifacts.com/output/job/88936b4b-94ca-4f6e-b059-4c9dd60b20a5/artifacts/0/dev/auto_examples/05_glm_second_level/plot_oasis.html#saving-model-outputs-to-disk

![image](https://github.com/nilearn/nilearn/assets/6961185/1b7b5e9a-703c-4f1c-9acf-062f9a581622)


## First level

### before

https://nilearn.github.io/dev/auto_examples/04_glm_first_level/plot_bids_features.html#saving-model-outputs-to-disk

![image](https://github.com/nilearn/nilearn/assets/6961185/3500807b-4f45-4d14-b74a-2923843002e5)


### after

https://output.circle-artifacts.com/output/job/88936b4b-94ca-4f6e-b059-4c9dd60b20a5/artifacts/0/dev/auto_examples/04_glm_first_level/plot_bids_features.html#only-generating-the-html-report

![image](https://github.com/nilearn/nilearn/assets/6961185/49d21e5e-d483-4d5b-8b61-40c4d2c5546d)

Reply:
So an increase for the 2nd level example.

For reference you can see the total time spent for succesfull doc build on main (usually between 1h15 and 1H45 minutes)

https://github.com/nilearn/nilearn/actions/workflows/build-docs.yml?query=branch%3Amain+is%3Asuccess

Reply:
See https://github.com/nilearn/nilearn/issues/4385